quotient-filter.c

```diff
@@ -508,19 +508,29 @@ bool qf_is_consistent(quotient_filter *qf) {
     // When entry counter is 0, make sure no elements contain in table
     // When entry counter isn't 0, make sure all elements' metabits is legal
     if (qf->entries == 0) {
-        for (uint64_t start = 0; start < qf->max_size; ++start)
-            if (get_elem(qf, start) != 0) return false;
-        return true;
+        // do a comparision per 8 bytes
+        size_t table_size = qf_table_size(qf->qbits, qf->rbits);
+        uint8_t *ptr;
+        for(ptr = (uint8_t *)qf->table; ptr < (uint8_t *)qf->table + table_size - 8; ptr+=8){
+            if (*(uint64_t *)ptr) return false;
+        }
+        if(table_size % 8){
+            uint64_t zeroblock = 0;
+            return !memcmp(ptr, &zeroblock, table_size % 8);
+        }
```

Comment:
you should add a `return ture;` after line 520.

Reply:
Already fixed it in next commit.